pages.es/common/du.md

```diff
@@ -1,28 +1,32 @@
 # du
 
-> Uso de disco: estima y resume el uso de espacio en disco de archivos y directorios.
+> Uso del disco: estima y resume el uso del espacio de archivos y directorios.
 > Más información: <https://www.gnu.org/software/coreutils/du>.
 
-- Lista los tamaños de un directorio y sus subdirectorios en las unidades dadas (B/KiB/MiB):
+- Lista los tamaños de un directorio y sus subdirectorios, en la unidad dada (B/KiB/MiB):
 
 `du -{{b|k|m}} {{ruta/al/directorio}}`
 
-- Lista los tamaños de un directorio y sus subdirectorios en formato legible para humanos (es decir, seleccionando automáticamente las unidades apropiadas para cada tamaño):
+- Lista los tamaños de un directorio y sus subdirectorios, de forma legible (es decir, seleccionando automáticamente la unidad adecuada para cada tamaño):
 
-`du -h {{ruta/al/directorio}}`
+`du -h {{ruta/a/directorio}}`
 
-- Muestra el tamaño de un solo directorio en formato legible para humanos:
+- Muestra el tamaño de un único directorio, en unidades legibles para el ser humano:
 
-`du -sh {{ruta/al/directorio}}`
+`du -sh {{ruta/a/directorio}}`
```

Comment:
let's unify {{ruta/al/directorio}}

Reply:
and remove "para el ser humano" to make it more terse, the other ones are more readable too :) .